src/components/Header/Header.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import useDebounce from "../../service/use-debounce";
+
+import { getFilm } from '../store/searchQuery';
+import { isModalOpen } from '../store/ModalSlice';
+import { isUserAuthoriz } from "../store/isUserAutherized";
+import { isUserAuth, searchFilms, authoriz, list, findfilm, lists } from '../store/selectors';
```

Comment:
```suggestion
import { isUserAuth, searchFilms, authoriz, list, findfilm, lists } from '../store/selectors';
```

isUserAuth - isUserAuthorized
authoriz ?? - authorize? might signIn?

what does it mean list?

searchFilms

findfilm ? findFilm findMovieById?

lists?